packages/shorebird_cli/lib/src/auth/auth.dart

```diff
@@ -111,7 +111,13 @@ class Auth {
     );
   }
 
-  Future<void> login(void Function(String) prompt) async {
+  /// Logs the string returned by [prompt] to the console and obtains auth
+  /// credentials. If [verifyEmail] is true, check that there is a Shorebird
+  /// user with the same email address as the newly-authenticated user.
+  Future<void> login(
+    void Function(String) prompt, {
+    bool verifyEmail = true,
```

Comment:
Thoughts on exposing a `signUp` instead of having `login` do both `login` and `signUp` based on a `bool`?